plot/plot_lmbench

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+
+plot() {
+data1="$1"
+data2="$2"
+data1_hostname="$3"
+data2_hostname="$4"
+gnuplot <<EOF
+set terminal png medium size 1920,1080
+set output "lmbench_result_combined.png"
+set title "Memory Latency Benchmark"
+
+set xlabel "Memory Depth (MB)"
+set ylabel "Latency (ns)"
+
+set grid
+
+set style line 1 lc rgb '#0060ad' lt 1 lw 2 pt 7 ps 1   # --- blue
+set style line 2 lc rgb '#ad6000' lt 1 lw 2 pt 7 ps 1   # --- red
+
+plot "${data1}" using 1:2 with linespoints ls 1 smooth unique title "${data1_hostname}", \
+	 "${data2}" using 1:2 with linespoints ls 2 smooth unique title "${data2_hostname}"
+
+EOF
+}
+
+if [ -n "$1" -a -n "$2" ]; then
+
+	DATA1_PATH="$( dirname "$(realpath $1)" )";
+	DATA2_PATH="$( dirname "$(realpath $2)" )";
```

Comment:
prefer to name as data1_path, similarly for later DATA2_HOSTNAME, can it use lower case?

Reply:
file submitted as another PR: https://github.com/intel/lkp-tests/pull/162